server/mux.go

```diff
@@ -98,7 +99,7 @@ func NewMux(opts MuxOpts, service Service) http.Handler {
 		return AuthorizedUser(
 			service.Store,
 			opts.UseAuth,
-			SuperAdminRoleName,
+			roles.SuperAdminRoleName,
```

Comment:
since SuperAdmin is not exactly a role, it's strange to see it referred to in this way. it is used like a role at some points of the codebase though.

Reply:
I agree. The reason this was added was that it allowed us to be explicit in `hasAuthorizedRole` in `auth.go`. Rather than relying on the default case, we could check explicitly.

```go
func hasAuthorizedRole(u *chronograf.User, role string) bool {
	switch role {
	case roles.ViewerRoleName:
		for _, r := range u.Roles {
			switch r.Name {
			case roles.ViewerRoleName, roles.EditorRoleName, roles.AdminRoleName:
				return true
			}
		}
	case roles.EditorRoleName:
		for _, r := range u.Roles {
			switch r.Name {
			case roles.EditorRoleName, roles.AdminRoleName:
				return true
			}
		}
	case roles.AdminRoleName:
		for _, r := range u.Roles {
			switch r.Name {
			case roles.AdminRoleName:
				return true
			}
		}
	case roles.SuperAdminRoleName:
		// SuperAdmins should have been authorized before this.
		// This is only meant to restrict access for non-superadmins.
		return false
	}

	return false
}
```

I agree it's kind of weird though. Happy to remove it if you think it only increases the confusion.

Reply:
Gotcha. What if we call it `HasSuperAdminStatus` or something to that effect?

Reply:
change `SuperAdminRoleName` to `SuperAdminStatus`

Reply:
fixed